packages/html-reporter/src/chip.tsx

```diff
@@ -53,17 +53,24 @@ export const AutoChip: React.FC<{
   noInsets?: boolean,
   children?: any,
   dataTestId?: string,
-  targetRef?: React.RefObject<HTMLDivElement>,
-}> = ({ header, initialExpanded, noInsets, children, dataTestId, targetRef }) => {
-  const [expanded, setExpanded] = React.useState(initialExpanded || initialExpanded === undefined);
-  return <Chip
-    header={header}
-    expanded={expanded}
-    setExpanded={setExpanded}
-    noInsets={noInsets}
-    dataTestId={dataTestId}
-    targetRef={targetRef}
-  >
-    {children}
-  </Chip>;
+  anchorId?: string,
```

Comment:
I find it strange that the first and only usage is inside `AutoChip`. I'd prefer to separate them for now, and wrap with `Anchor` where needed to separate concerns.

Reply:
If we want the chip to auto-expand on reveal, then we need to combine them. If not, then i'm happy to separate.